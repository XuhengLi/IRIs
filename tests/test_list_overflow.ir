int main()
    int[] a
    int b
    [1,2,3]
    |
    a
    a[3]
    |
    b
    printi(b)
Siri
