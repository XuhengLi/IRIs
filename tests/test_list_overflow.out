IRIS error: Bad address
0
